tests/tests_integration/test_api/test_geospatial.py

```diff
@@ -0,0 +1,410 @@
+        res = COGNITE_CLIENT.geospatial.search_features(
+            feature_type=test_feature_type, filter={}, order_by=[OrderSpec(property="temperature", direction="ASC")]
+        )
+        assert res[0].temperature == -10.8
+        assert res[1].temperature == 12.4
+
+        res = COGNITE_CLIENT.geospatial.search_features(
+            feature_type=test_feature_type, filter={}, order_by=[OrderSpec(property="temperature", direction="DESC")]
+        )
+        assert res[0].temperature == 12.4
+        assert res[1].temperature == -10.8
+
+    def test_update_feature_types(self, test_feature_type):
+        res = COGNITE_CLIENT.geospatial.update_feature_types(
+            update=FeatureTypeUpdate(
+                external_id=test_feature_type.external_id,
+                add=PropertyAndSearchSpec(
+                    properties={"altitude": {"type": "DOUBLE", "optional": True}},
+                    search_spec={
+                        "altitude_idx": {"properties": ["altitude"]},
+                        "pos_alt_idx": {"properties": ["position", "altitude"]},
+                    },
+                ),
+            )
+        )
+        assert len(res) == 1
+        assert len(res[0].properties) == 8
+        assert len(res[0].search_spec) == 6
+
+    def test_stream_features(self, large_feature_type, many_features):
+        features = COGNITE_CLIENT.geospatial.stream_features(feature_type=large_feature_type, filter={})
+        feature_list = FeatureList(list(features))
+        assert len(feature_list) == len(many_features)
+
+    def test_to_pandas(self, test_feature_type, two_test_features):
+        df = two_test_features.to_pandas()
+        assert list(df) == [
+            "externalId",
+            "position",
+            "volume",
+            "temperature",
+            "pressure",
+            "createdTime",
+            "lastUpdatedTime",
+        ]
+
+    def test_to_geopandas(self, test_feature_type, two_test_features):
+        gdf = two_test_features.to_geopandas(geometry="position")
+        assert list(gdf) == [
+            "externalId",
+            "position",
+            "volume",
+            "temperature",
+            "pressure",
+            "createdTime",
+            "lastUpdatedTime",
+        ]
+        geopandas = utils._auxiliary.local_import("geopandas")
+        assert type(gdf.dtypes["position"]) == geopandas.array.GeometryDtype
+
+    def test_from_geopandas(self, test_feature_type, two_test_features):
+        gdf = two_test_features.to_geopandas(geometry="position")
+        fl = FeatureList.from_geopandas(test_feature_type, gdf)
+        assert type(fl) == FeatureList
+        assert len(fl) == 2
+        for f in fl:
+            for attr in test_feature_type.properties.items():
+                attr_name = attr[0]
+                if attr_name.startswith("_"):
+                    continue
+                assert hasattr(f, attr_name)
```

Comment:
These tests should be moved to `tests/tests_unit/test_api/test_geospatial.py`. Then you will need to mark them with `@pytest.mark.dsl` so they are excluded from the core tests (which tests all functionality which doesn't require extra dependencies such as pandas/geopandas)